.travis.yml

```diff
@@ -1,24 +1,34 @@
-dist: trusty
+dist: xenial
 sudo: required
-cache: cargo
```

Comment:
Given that [the docs say](https://docs.travis-ci.com/user/caching/#rust-cargo-cache) "[cache: cargo] caches $HOME/.cargo and $TRAVIS_BUILD_DIR/target" and the new value of `cache` below, you effectively removed `target` and `.cargo/registry` from the cache.

Comparing [a build log before these changes](https://travis-ci.org/andygrove/sqlparser-rs/builds/540583671?utm_source=github_status&utm_medium=notification) and [a build with the new config](https://travis-ci.org/andygrove/sqlparser-rs/builds/540865573?utm_source=github_status&utm_medium=notification), the downloading and re-building takes more time with the newer build, as expected. I suppose there's a benefit to not accumulating the dead weight in the cache? Care to clarify?

Reply:
Yeah, at some point the scale will tip in favor of a clean `cargo build`, as downloading the cache will take longer than just compiling from scratch. I'm happy to kick that can down the road, though, since I think you're right that, at the moment, the scales are firmly in favor of using the default `cache: cargo` settings. I've updated the PR accordingly.